style/style.css

```diff
@@ -82,4 +106,153 @@ body {
     justify-content: center;
     margin-top: 10px;
   }
+
+  .main2 {
+    display: grid;
+    grid-row-gap: 10px;
+  }
+
+  .title {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    margin: 20px;
+  }
+
+  .title h1 {
+    font-size: 40px;
+    font-weight: 700;
+    padding: 10px;
+    color: #172b4d;
+    text-align: center;
+  }
+
+  .title .end-line {
+    background-color: #36b37e;
+    width: 48px;
+    height: 4px;
+    margin: 30px;
+    border-radius: 20px;
+  }
+
+  .project-container {
+    margin-left: 10px;
+    margin-right: 10px;
```

Comment:
- [ ] let's make sure we follow a spacing system of Figma design, we need to increase instead of `10px` to match the Figma design As explained in the Loom video how to get exact space we should nail this 😎 

Your Impelementation | Figma desgin 
:--:|:--:
![image](https://user-images.githubusercontent.com/87689547/148624883-9853caeb-c614-4e97-bbb1-547b22daa049.png) | ![image](https://user-images.githubusercontent.com/87689547/148624906-dc06371a-264c-4342-9bec-5211b8c492ac.png)

Reply:
I set them 24px as indicated in the Figma design, Thanks 🙏